app/build.gradle.kts

```diff
@@ -113,15 +116,21 @@ fun DependencyHandlerScope.globalTestImplementation(dep: Any) {
 }
 
 dependencies {
+    testImplementation("org.json:json:20160810")
```

Comment:
Improvement: use version catalog instead (check the Android Studio suggestion)

Reply:
I fixed it. Thank you!